src/blocks/weather.rs

```diff
@@ -331,6 +382,34 @@ fn convert_wind_direction(direction_opt: Option<f64>) -> &'static str {
     }
 }
 
+// Compute the average wind speed and direction
+fn average_wind(winds: &Vec<Wind>) -> Wind {
+    let mut opposite = 0.0;
+    let mut adjacent = 0.0;
```

Comment:
I think "north" and "east" are more descriptive

Reply:
The function seems correct, but you swapped "north" and "east". When angle is 0, the direction is north, but in this function it is east.

Reply:
I think the function looks right... I'll add some tests, but it seems like if you average the wind at 0 degrees the result is also at 0 degrees, not 90 degrees (East)

Reply:
The function result _is_ correct, I was referring to local variables "north" and "east". They are supposed to represent the components of the wind, so when the direction is 0 degrees and speed is 1, `north` should be `1.0` and `east` should be `0.0`. Right now it's the opposite.

Reply:
🤦‍♂️